tic-tac-toe.js

```diff
@@ -1,7 +1,71 @@
-function TicTacToe() {
+var turn = 0;
+var icon = 'X';
+var player = 'Player 1';
 
+function TicTacToe() {
+  this.board = ["cell 0", "cell 1","cell 2","cell 3","cell 4","cell 5","cell 6","cell 7","cell 8"];
 }
 
-TicTacToe.prototype = {
-  
-}
+
+
+TicTacToe.prototype.getBoard = function() {
+  return this.board;
+};
+
+TicTacToe.prototype.markBoard = function() {
+  var self = this;
+  $("td").bind("click", function() {
+    var index = 0;
+    if (turn%2 === 0){
+      //var index = 0;
+        for (index = 0; index < self.board.length; index ++){
+          if (self.board[index] === this.className){
```

Comment:
Does `#className` return all classes, or just the first one? For example, you have both "cell" and "0" as classes on a single cell.